bootcamp-milestone-2/src/app/globals.css

```diff
@@ -0,0 +1,273 @@
+    text-decoration: underline;
+}
+
+.navbar a:active {
+    color: var(--cg-orange);
+}
+
+.logo a {
+    font-family: Georgia;
+}
+
+.logo a:hover {
+    text-decoration: None;
+}
+
+main {
+    padding: 0 20px 40px 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.page-title {
+    display: flex;
+    justify-content: center;
+    font-size: 40px;
+    font-family: Georgia;
+}
+
+/* CSS for portfolio grid */
+
+.project {
+    display: flex;
+    flex-direction: row;
+    max-width: 100%;
+    margin: 1em 0 2em 0;
+
+    /* Debug border */
+    /* border: 2px solid blue; */
+}
+
+.item-link {
+    width: 70%;
+
+    /* Debug border */
+    /* border: 2px solid red; */
+}
+
+.project-image {
+    object-fit: cover;
+    width: 100%;
+    height: 500px;
+    flex-basis: 50px;
+    flex-grow: 2;
+
+    /* Debug border */
+    /* border: 2px solid black; */
+}
+
+.project-details {
+    display: flex;
+    background-color: var(--cg-white);
+    width: 30%;
+    height: 500px;
+    flex-shrink: 1;
+    padding: 0 20px;
+    margin: 0px;
+    line-height: 1.5;
+    overflow: scroll;
+
+    flex-direction: column;
+    text-align: center;
+    justify-content: center;
+
+    /* Debug border */
+    /* border: 2px solid green; */
+}
+
+.project-details a {
+    color: var(--cg-cyan);
+    text-decoration: none;
+    font-weight: bold;
+    letter-spacing: 1px;
+}
+
+.project-details a:hover {
+    color: var(--cg-orange);
+    text-decoration: underline;
+}
+
+.project-details {
+    font-weight: lighter;
+}
+
+.project-name {
+    font-weight: bold;
+    font-size: 20px;
+}
+
+/* CSS for home page */
+
+.about {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 20px;
+    background-color: #f8f8f8;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    /* box-shadow: 0 0 10px rgba(0, 0, 0, 0.1); */
+}
+
+.about-image img {
+    max-width: 300px;
+    border-radius: 3px;
+    /* box-shadow: 0 0 10px rgba(0, 0, 0, 0.2); */
+}
+
+.about-text {
+    flex: 1;
+    margin-left: 20px;
+}
+
+.about-text h2 {
+    font-size: 24px;
+    margin-bottom: 10px;
+}
+
+.about-text p {
+    font-size: 16px;
+    line-height: 1.5;
+}
+
+/* CSS for contact form */
+
+label {
+    display: block;
+    margin-bottom: 8px;
+    font-family: Georgia;
+    font-weight: bold;
+}
+
+input[type="text"],
+input[type="email"],
+textarea {
+    width: 100%;
+    padding: 10px;
+    margin-bottom: 15px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    font-size: 16px;
+    resize: none;
+}
+
+/* Add some spacing to the form */
+form {
+    width: 500px;
+    margin: 0 auto;
+}
+
+
+/* CSS for contact submit button */
+#submitButtonContainer {
+    text-align: center;
+}
+
+#submitButton {
+    background-color: lightblue;
+    border: none;
+    padding: 10px 20px;
+    border-radius: 30px;
+    cursor: pointer;
+}
+
+body {
+    color: white;
+}
```

Comment:
you already have styling for the body in the same css file, you can change its color above.